internal/examples/unittest/time.go

```diff
@@ -0,0 +1,7 @@
+package unittest
+
+import "github.com/tchajed/goose/machine"
+
+func sleep() {
+	machine.Sleep(1 * 1000000000)
```

Comment:
Doesn't seem worth to have the test sleep an entire second, can you reduce this?

Reply:
Yes, done.

Reply:
Thanks!